cppapi/server/idl/CMakeLists.txt

```diff
@@ -8,16 +8,38 @@ message("Generate tango.h, tangoSK.cpp and tangoDynSK.cpp from idl")
 message("Using OMNIIDL_PATH=${OMNIIDL_PATH}")
 message("Using IDL=${IDL_PKG_INCLUDE_DIRS}")
 
+MESSAGE( STATUS "OMNIIDL_PATH: " ${OMNIIDL_PATH} )
+
+ if(WIN32)
+ set_source_files_properties(tangoDynSK.cpp PROPERTIES COMPILE_DEFINITIONS _OMNIORB_STUB_DLL)
+ set_source_files_properties(tangoDynSK.cpp PROPERTIES COMPILE_DEFINITIONS __NT__)
+ set_source_files_properties(tangoSK.cpp PROPERTIES COMPILE_DEFINITIONS _OMNIORB_STUB_DLL)
+ set_source_files_properties(tangoSK.cpp PROPERTIES COMPILE_DEFINITIONS __NT__)
+        if(${CMAKE_SIZEOF_VOID_P} EQUAL 8)
+execute_process(COMMAND ${OMNIIDL_PATH}/omniidl.exe -I${IDL_PKG_INCLUDE_DIRS} -bcxx -Wbh=.h -Wbs=SK.cpp -Wbd=DynSK.cpp -Wba ${IDL_PKG_INCLUDE_DIRS}/tango.idl
+                WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+                RESULT_VARIABLE FAILED)
+        else()
+execute_process(COMMAND ${OMNIIDL_PATH}/omniidl.exe -I${IDL_PKG_INCLUDE_DIRS} -bcxx -Wbh=.h -Wbs=SK.cpp -Wbd=DynSK.cpp -Wba ${IDL_PKG_INCLUDE_DIRS}/tango.idl
```

Comment:
It looks like there is no difference between what is executed here when (${CMAKE_SIZEOF_VOID_P} EQUAL 8) and in the else.
It looks like this if could be removed and this code simplified.

Reply:
Surely.

I prepared this in case the 64 omniidl could be different.

Reply:
Since it appears this is not different, could you please remove this if(${CMAKE_SIZEOF_VOID_P} EQUAL 8)?
Thanks! :-)